packages/cadl-ranch-specs/http/azure/core/traits/mockapi.ts

```diff
@@ -17,8 +17,15 @@ Scenarios.Azure_Core_Traits_smokeTest = passOnSuccess(
       throw new ValidationError("Expected path param id=1", "1", req.params.id);
     }
     req.expect.containsHeader("foo", "123");
-    req.expect.containsHeader("if-match", '"valid"');
-    req.expect.containsHeader("if-none-match", '"invalid"');
+    const if_none_match = req.headers["if-none-match"];
+    const if_match = req.headers["if-match"];
+    if (if_none_match !== '"invalid"' && if_match !== '"valid"') {
+      throw new ValidationError(
+        `Expected header "if-none-match" equals "invalid" but got ${if_none_match} or "if-match" equals "valid" but got ${if_match}`,
+        if_none_match,
+        if_match,
```

Comment:
parameter is wrong, I think one is "expected", another is "actual"

Reply:
updated:
![image](https://github.com/Azure/cadl-ranch/assets/70930885/2c0e99af-d2fb-483d-93a1-fc58d52deb42)